Orange/projection/base.py

```diff
@@ -145,7 +145,7 @@ def proj_variable(i, name):
     def _get_var_names(self, n):
         postfixes = ["x", "y"] if n == 2 else [str(i) for i in range(1, n + 1)]
         names = [f"{self.var_prefix}-{postfix}" for postfix in postfixes]
-        return get_unique_names(self.orig_domain, names)
+        return names
```

Comment:
Since `DomainProjection` can be used independently of any widget, I'd leave it as it was (retain the `get_unique_names`)